pom.xml

```diff
@@ -16,8 +17,19 @@
         <maven.checkstyle.plugin.configLocation>
             https://raw.githubusercontent.com/mate-academy/style-guides/master/java/checkstyle.xml
         </maven.checkstyle.plugin.configLocation>
+        <maven.compiler.source>11</maven.compiler.source>
```

Comment:
do we need these lines?

Reply:
No, I was debugging my solution and thought it will not work without it.